line-sdk/build.gradle

```diff
@@ -82,10 +82,10 @@ dependencies {
     testImplementation 'org.robolectric:robolectric:4.3'
     testImplementation 'org.mockito:mockito-core:2.28.2'
 
-    api 'io.jsonwebtoken:jjwt-api:0.10.5'
-    implementation 'io.jsonwebtoken:jjwt-impl:0.10.5'
-    implementation('io.jsonwebtoken:jjwt-orgjson:0.10.5') {
-        exclude group: 'org.json', module: 'json' //provided by Android natively
+    api "io.jsonwebtoken:jjwt-api:0.11.1"
```

Comment:
please use single quote as other lines.

Reply:
extracted jjwt_version  as a property in build script